polymath/extensions/pickler.py

```
@@ -90,920 +90,930 @@
-        references = 'fpzip'
+        return tuple(new_digits)
 
-    if isinstance(references, list):
-        references = tuple(references)
+    @staticmethod
+    def _validate_pickle_reference(references):
+        """Validate and return the pickle reference values."""
 
-    elif not isinstance(references, tuple):
-        references = (references, references)
+        original_references = references    # Flake8 thinks this variable is unused # noqa
```

Ruff (0.14.7)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)